lib/fog/core/model.rb

```diff
@@ -21,16 +21,7 @@ def initialize(new_attributes = {})
     end
 
     def inspect
-      Thread.current[:formatador] ||= Formatador.new
-      data = "#{Thread.current[:formatador].indentation}<#{self.class.name}"
-      Thread.current[:formatador].indent do
-        unless self.class.attributes.empty?
-          data << "\n#{Thread.current[:formatador].indentation}"
-          data << self.class.attributes.map { |attribute| "#{attribute}=#{send(attribute).inspect}" }.join(",\n#{Thread.current[:formatador].indentation}")
-        end
-      end
-      data << "\n#{Thread.current[:formatador].indentation}>"
-      data
+      Fog::Formatador.new(self)
```

Comment:
Yeah. I think I would be fine with a module. There are potential indentation + threads issues, but it seems unlikely we would run in to them here. Maybe a second argument for nested stuff, but perhaps as a hash of options, like `Fog::Formatador.inspect(self, :include_nested => false)` or something like that. Buys us a bit more flexibility as I suspect that might not be the only option we would think of as we centralize more stuff.